CoreFoundation/Base.subproj/CFBase.h

```diff
@@ -80,6 +80,11 @@
 #if !defined(__MACTYPES__)
 #if !defined(_OS_OSTYPES_H)
 #if DEPLOYMENT_RUNTIME_SWIFT
+
+#ifdef __cplusplus
+#define _Bool bool
+#endif
```

Comment:
I wonder if we want to use `stdbool.h` here as well as we do on the other platforms.

Reply:
Yeah, that seems like a better approach - using the standard to use the proper spelling irrespective of the language.

Reply:
Are there any other uses of `_Bool` in CF?

Reply:
This is the list I have. I just need to find a cycle or two to go through and change them and run the tests.

```
./CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h:static inline _Bool _resizeConditionalAllocationBuffer(_ConditionalAllocationBuffer *_Nonnull buffer, size_t amt) {
./CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h:static inline _Bool _withStackOrHeapBuffer(size_t amount, void (__attribute__((noescape)) ^ _Nonnull applier)(_ConditionalAllocationBuffer *_Nonnull)) {
./CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h:static inline _Bool _withStackOrHeapBuffer(size_t amount, void (__attribute__((noescape)) ^ _Nonnull applier)(_ConditionalAllocationBuffer *_Nonnull)) {
./CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h:static inline _Bool _withStackOrHeapBufferWithResultInArguments(size_t amount, void (__attribute__((noescape)) ^ _Nonnull applier)(void *_Nonnull memory, size_t capacity, _Bool onStack)) {
./CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h:static _Bool const _CFHasRenameat2 = 1;
./CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h:static _Bool const _CFHasRenameat2 = 0;
./CoreFoundation/Base.subproj/CFBase.h:    typedef _Bool                   Boolean;
./CoreFoundation/Base.subproj/ForFoundationOnly.h:#define BOOL _Bool
./CoreFoundation/Base.subproj/CFOverflow.h:    static _Bool __os_warn_unused(_Bool x) __attribute__((__warn_unused_result__));
./CoreFoundation/Base.subproj/CFOverflow.h:    static _Bool __os_warn_unused(_Bool x) { return x; }
./Darwin/shims/FoundationOverlayShims.h:    _Bool onStack;
./Darwin/shims/FoundationOverlayShims.h:static inline _Bool _resizeConditionalAllocationBuffer(_ConditionalAllocationBuffer *_Nonnull buffer, size_t amt) {
./Darwin/shims/FoundationOverlayShims.h:static inline _Bool _withStackOrHeapBuffer(size_t amount, void (__attribute__((noescape)) ^ _Nonnull applier)(_ConditionalAllocationBuffer *_Nonnull)) {
./CoreFoundation/PlugIn.subproj/CFBundle_Grok.c:static inline _Bool _CF_isspace(int c) {
```